docs/BEST_PRACTICES.md

```diff
@@ -1,13 +1,4 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  
-<!-- *generated with [DocToc](https://github.com/thlorenz/doctoc)* -->
-
-- [Deployment best practices](#deployment-best-practices)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
-#### Deployment best practices
+## Deployment best practices
```

Comment:
What about HMAC? Should we also mention it here?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/566/commits/90a4bc978833138549ea948eb08122027fa9d843

>